src/main/java/com/todoist/pojo/Utils.java

```diff
@@ -23,6 +25,15 @@ static int clamp(int val, int min, int max) {
         }
     }
 
+    static <K, V> Map<K, V> unmodifiableMap(Map<K, V> m) {
```

Comment:
Unsure why in `unmodifiableSet` we use `s` instead of `set`, but maybe let's change that and use `map` here (instead of `m`) as well?